recipes/ls-gkm/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: ls-gkm
+  version: 0.0.1
+
+source:
+  fn: ls-gkm.zip
+  url: https://github.com/Dongwon-Lee/lsgkm/archive/master.zip
```

Comment:
Can you please also use this URL?
https://github.com/Dongwon-Lee/lsgkm/archive/164a4a48f6387e67b5d955db932f8a403de6c321.zip

Otherwise we will get a new version every time we rebuild this package.

Reply:
yes. sure. Sorry was not ready yet but did already the push..